### ISSUE TITLE
Light State "Blinking" not being carried over from 10.7 table, resulting in black lights

### ISSUE THREAD

**Issue by gjupp** [vpinball/vpinball#585]
**Describe the bug**
With a table created in pre 10.8 vpx that has lights with "Blinking" state attribute set, this attribute is being ignored in 10.8 and the Light/State field either appears empty or else needs to be changed to "off" and then back to "Blinking" again in order to be set correctly (some weirdness went on when I was adjusting this so I can't be sure, hence why I have said "either" as with some lights the State field was empty, but some others had "blinking" state but still suffered the black light issue).

This results in black circles where they should be lit. Several other users at VPU confirmed same issue.

**To Reproduce**
Download Robowar table https://vpuniverse.com/files/file/14792-robo-war-premier-1988
Play and note soft edged black circles on GI lights on Apron area.
Can be easily fixed in game by setting those Light States to "On" (as discovered by user trizoneGB at VPU.)

**Expected behavior**
Lights should be lit, not black circles. In this particular case the table creator has set the lights to blinking but I'm not sure why as they don't seem to actually blink on VPX 10.74 but just remain lit. Perhaps there is some point in the game where they do actually blink? Setting them back to "Blinking" in VPX 10.8 makes them blink at a rapid speed which does not seem correct and is hard on the eyes.

**Versions & Settings**
 VPinballX-10.8.0-1272-af469be-Release-win-x86

![robowar](https://github.com/vpinball/vpinball/assets/95056536/6933e622-6912-4f8c-b262-8be0789a7121)

**Comment by Smilelikejoker** [comment]
Ok... I'm experiencing this same bug.  However, I can create the bug all day long in 10.8 (32bit and 64bit) by merely opening and closing the table in the editor several times (without closing VPX).  Sometimes the bulbs have the 'Blinking' state and others times they are blank.  Regardless of the state of the bulbs in the editor, the bulbs default to no state if you actually start the table.

NOTE:  There are a few tables you can do this on right now easily.

This is how I am reproducing:
- Download Gold Wings table https://vpuniverse.com/files/file/14805-gold-wings-gottlieb-1986/  or table listed above
- Open VPX 10.8 DX Rev 1271 (either 32 bit or 64 bit)
- Close the auto pick table box
- Go to File and open table that way.  Do not launch the table.  Stay in the Editor.
- You will see 3 large GI bulbs at the bottom of the table.  Click each one and check their state.  2 of the 3 bulbs might be set to Blinking or not have a state.
- Close the table without closing VPX by using the little X.
- Go to File and reopen the table and check bulbs again.  You may have to do this 2 or 3 times, but you will see the bug, at least visually.
- If you actually start the table, the 2 GI bulbs default to no state regardless of what the editor shows.  When you go to close the table now, it wants you to save it as if you made a change (but you haven't).
- In some rare occasions, VPX might just crash when you select a bulb to view (not sure if related).

This is a weird one since VPX flip flops on these states in the editor, but is ignored if the table is loaded.  Regardless, the above method is the best I've found to reproduce the bug both visually in the editor and when you load the table.

VPX 10.8 DX Rev 1271 (x32 and x64)
Windows 11 Pro Ver 22H2
NVidia 3080TI Driver 536.23
HDR enabled on TV and in Windows
[VPinballX.zip](https://github.com/vpinball/vpinball/files/11813970/VPinballX.zip)
![vidsetting](https://github.com/vpinball/vpinball/assets/102932021/5fe9c19b-91c4-4b0f-9b0e-098378ac1586)

**Comment by Smilelikejoker** [comment]
Additional Note - Manually changing the bulbs to the blinking state and then saving fixes the issue.  I cannot reproduce after saving the table if manually fixed.

**Comment by gjupp** [comment]
@Smilelikejoker , yes this flipflopping between empty state and blinking state was also what I experienced and attempted to badly explain in my post, your description is a little clearer. I also wasn't sure if I was remembering what happened correctly but seems I was.

I also experienced a VPX crash as you described by attempting to click one of the lights when I had lots of layers visible.

**Comment by Smilelikejoker** [comment]
I figured I'd test some of these tables with bulb issues in 10.74 DX Rev 410 for my own curiosity.  Tables load without any issues.

I've noticed another issue with one of these tables that has the bulb state issue.  Unfortunately, I can't seem to fix it in 10.8 Rev 1271 no matter what I do.  It randomly fixes itself at times, but then reverts back to the bug.  I thought it might be related to the camera as playing around with it has fixed it, but not in a way I can pinpoint or constantly repeat.  See image:
![weirdline](https://github.com/vpinball/vpinball/assets/102932021/2bd232d9-b0c0-4d15-81f5-ce282bddfc71)
Table is Who Dunnit (https://vpuniverse.com/files/file/14804-who-dunnit-bally-1995/ ).  This issue could need its own bug page though unless it is somehow related to how VPX is treating these particular tables.

**Comment by gjupp** [comment]
@Smilelikejoker yeah I also noticed that today. It seems to be an issue with Primitive43 on layer_1. If you go to camera edit mode and adjust table inclination you can see that the shadow line moves up and down and is due to the primitive floating very high, positioned well above the table surface for some reason.

**Comment by Smilelikejoker** [comment]
Interesting discussion happened on the Virtual Pinball Chat Discord server.  User movieguru stated that he is on 10.8 BETA 1 Rev 1138 x64 and does not have the issue on these particular tables like everyone else has.  He stated that he did not have to make any corrections to the bulbs and they all loaded fine from the start.

In the interest of testing, I switched to that revision with a fresh table and still had the issue.  I went through adjusting every setting off and on and even messed with my video card driver settings.  I could not get these table to display correctly at all.

If movieguru is indeed correct, it might be worth reaching out to him on Discord to get more details about his full setup.  There has to be something different about it for him to not have the issue.
